packages/abstractions/src/apiError.ts

```diff
@@ -1,16 +1,21 @@
-/** Parent interface for errors thrown by the client when receiving failed responses to its requests. */
```

Comment:
for my education: are doc comment ignored for interfaces?

Reply:
I don't understand the context of your question @baywet

Reply:
I saw that you removed the doc comment, can you explain why?

Reply:
Oh, I see! I didn't remove that intentionally; it must've slipped out during one of the merges. I'll revert it too.